utils/time.c

```diff
@@ -21,11 +21,12 @@
 
 uint64_t current_timestamp_ms() {
 #ifdef _WIN32
-  SYSTEMTIME time;
+  __int64 wintime;
 
-  GetSystemTime(&time);
-  LONG ms = (time.wSecond * 1000) + time.wMilliseconds;
-  return ms;
+  GetSystemTimeAsFileTime((FILETIME*)&wintime);
```

Comment:
Do not cast a pointer to a FILETIME structure to either a ULARGE_INTEGER* or __int64* value because it can cause alignment faults on 64-bit Windows.
ref: https://docs.microsoft.com/en-us/windows/desktop/api/minwinbase/ns-minwinbase-filetime

Reply:
I'm doing the opposite here.
Even though they don't specify anything about that and it appears to be ok (https://stackoverflow.com/questions/5404277/porting-clock-gettime-to-windows#comment64618848_31335254), I'll try to remove ambiguity.